be/src/vec/exec/scan/scanner_scheduler.cpp

```diff
@@ -123,9 +123,7 @@ Status ScannerScheduler::init(ExecEnv* env) {
             config::doris_scanner_thread_pool_queue_size, "local_scan");
 
     // 3. remote scan thread pool
-    _remote_thread_pool_max_size = config::doris_max_remote_scanner_thread_pool_thread_num != -1
-                                           ? config::doris_max_remote_scanner_thread_pool_thread_num
```

Comment:
`std::max(512, CpuInfo::num_cores() * 10)`
this value is for object storage, think it twice.